modules/hydrodyn/src/HydroDyn_Input.f90

```diff
@@ -2398,7 +2408,23 @@ SUBROUTINE HydroDynInput_ProcessInitData( InitInp, ErrStat, ErrMsg )
       END IF
    END IF
 
-
+      ! Linearization Checks
+   ! LIN-TODO:
```

Comment:
Are these checks not needed anymore?

Reply:
ok, it seems to be done below

Reply:
These are part of the changes I started with from #350.